dev/NavigationView/NavigationView.xaml

```diff
@@ -163,7 +163,15 @@
                                     Style="{TemplateBinding PaneToggleButtonStyle}"
                                     AutomationProperties.LandmarkType="Navigation"
                                     Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.PaneToggleButtonVisibility}"
-                                    VerticalAlignment="Top"/>
+                                    VerticalAlignment="Top">
+                                    <TextBlock
+                                        x:Name="PaneTitleTextBlock" 
+                                        Grid.Column="0"
+                                        Text="{TemplateBinding PaneTitle}"
+                                        HorizontalAlignment="Left"
+                                        VerticalAlignment="Center"
+                                        Style="{StaticResource NavigationViewItemHeaderTextStyle}"/>
```

Comment:
Won't placing the TextBlock inside the button cause all sorts of strange hover and focus visuals when the title is not an empty string?

Reply:
I had not realized that, without your change, the hamburger button's footprint grows to the full width of the pane when PaneTitle is not empty. Not necessarily the design choice I would have made, but you're not changing it. Please proceed :)